mycroft/messagebus/client/ws.py

```diff
@@ -40,7 +40,28 @@ class WebsocketClient(object):
     def __init__(self, host=client_config.get("host"),
                  port=client_config.get("port"),
                  path=client_config.get("route"),
-                 ssl=str2bool(client_config.get("ssl"))):
+                 ssl=None):
+
+        if not host:
+            raise ValueError("Missing or empty host in mycroft.ini "
+                             "[messagebus_client] section")
+
```

Comment:
Please, move all those validations to a new method to make the init logic clear. In addition, as their messages are exactly the same you can reuse then in a common method which receives the property name as a parameter e.g.:
```
def validate_param(value, name)
  if not value:
    raise ValueError("Missing or empty %s in mycroft.ini [messagebus_client] section", name)
```